packages/mobile/src/analytics/Properties.tsx

```diff
@@ -273,6 +273,10 @@ interface VerificationEventsProperties {
     issuer: any
     error: string
   }
+  [VerificationEvents.verification_reveal_attestation_status]: {
+    issuer: any
+    status: any
```

Comment:
ok, chatted with Tim.  Can we make this event reflect the fields that the attestation service returns?  iow, just parse the first level of the JSON response field-for-field, and then the parsing/transformation of the individual errors should be doable in our metrics pipeline.

```
  issuer: any,
  identifier: any,
  success: boolean,
  account:  any,
  attempt: number,
  countryCode: string,
  status: string,
  provider: string,
  duration: number,
  errors: any
```

^ though, I think some of these already are automatically added as generic Valora events like 'identifier' and 'account' but not sure where that happens